lib/classes/package_builder.rb

```diff
@@ -0,0 +1,14 @@
+class PackageBuilder
```

Comment:
```suggestion
class PackageDetailBuilder
```
Seems more appropriate, since it doesn't return a Package

Reply:
yeah, right, but we will be moving the package record building process to this class in future, so name it `PackageBuilder`. Will change that.

Reply:
Maybe it can be `PackageDetailBuilder` with just a `build()` method.
And later when you create `PackageBuilder`, it can just call `PackageDetailBuilder` : )